File: 01668.cpp
#include <iostream>
using namespace std;

int* trophy;

int main() {
	int N; cin >> N;

	trophy = new int[N];
	for (int i = 0; i < N; i++)
		cin >> trophy[i];

	int cnt = 0, max_trophy = 0;
	for (int i = 0; i < N; i++) {
		if (trophy[i] > max_trophy) {
			cnt++;
			max_trophy = trophy[i];
		}
	}
	cout << cnt << endl;

	cnt = 0, max_trophy = 0;
	for (int i = N - 1; i >= 0; i--) {
		if (trophy[i] > max_trophy) {
			cnt++;
			max_trophy = trophy[i];
		}
	}
	cout << cnt << endl;

	delete[] trophy;
}

File: 01668.py
N = int(input())

trophy = []
for i in range(N):
    trophy.append(int(input()))

cnt = 0; max_trophy = 0
for h in trophy:
    if h > max_trophy:
        cnt += 1
        max_trophy = h
print(cnt)

cnt = 0; max_trophy = 0
for i in range(N - 1, -1, -1):
    if trophy[i] > max_trophy:
        cnt += 1
        max_trophy = trophy[i]
print(cnt)
